### ISSUE TITLE
conda package generation failure late July 2022

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#1218]
See https://github.com/robotology/robotology-superbuild/actions/runs/2736174583 .
~~~
2022-07-26T02:25:38.2375034Z Mamba failed to solve:
2022-07-26T02:25:38.2376084Z  - eigen
2022-07-26T02:25:38.2376956Z  - xorg-libxfixes
2022-07-26T02:25:38.2377453Z  - qpoases
2022-07-26T02:25:38.2377965Z  - libstdcxx-ng >=12
2022-07-26T02:25:38.2384348Z  - yarp-cxx
2022-07-26T02:25:38.2388300Z  - blockfactory
2022-07-26T02:25:38.2388713Z  - osqp-eigen
2022-07-26T02:25:38.2388984Z  - idyntree
2022-07-26T02:25:38.2389271Z  - libgcc-ng >=12
2022-07-26T02:25:38.2389578Z  - icub-main
2022-07-26T02:25:38.2389725Z 
2022-07-26T02:25:38.2389824Z with channels:
2022-07-26T02:25:38.2389974Z 
2022-07-26T02:25:38.2390090Z The reported errors are:
2022-07-26T02:25:38.2390474Z - Encountered problems while solving:
2022-07-26T02:25:38.2391101Z -   - package idyntree-4.0.0-py36hd3cff06_0 requires ipopt >=3.14.1,<3.14.2.0a0, but none of the providers can be installed
2022-07-26T02:25:38.2397891Z - 
2022-07-26T02:25:38.2444054Z Traceback (most recent call last):
2022-07-26T02:25:38.2444782Z   File "/usr/share/miniconda/envs/test/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 124, in mamba_get_install_actions
2022-07-26T02:25:38.2445253Z     solution = solver.solve_for_action(_specs, prefix)
2022-07-26T02:25:38.2445861Z   File "/usr/share/miniconda/envs/test/lib/python3.8/site-packages/boa/core/solver.py", line 226, in solve_for_action
2022-07-26T02:25:38.2446252Z     t = self.solve(specs)
2022-07-26T02:25:38.2447248Z   File "/usr/share/miniconda/envs/test/lib/python3.8/site-packages/boa/core/solver.py", line 216, in solve
2022-07-26T02:25:38.2447849Z     raise RuntimeError("Solver could not find solution." + error_string)
2022-07-26T02:25:38.2448573Z RuntimeError: Solver could not find solution.Mamba failed to solve:
2022-07-26T02:25:38.2448917Z  - eigen
2022-07-26T02:25:38.2449174Z  - xorg-libxfixes
2022-07-26T02:25:38.2449412Z  - qpoases
2022-07-26T02:25:38.2449675Z  - libstdcxx-ng >=12
2022-07-26T02:25:38.2450184Z  - yarp-cxx
2022-07-26T02:25:38.2450614Z  - blockfactory
2022-07-26T02:25:38.2450884Z  - osqp-eigen
2022-07-26T02:25:38.2451145Z  - idyntree
2022-07-26T02:25:38.2451413Z  - libgcc-ng >=12
2022-07-26T02:25:38.2451663Z  - icub-main
~~~

In a nutshell, the problem is that the conda-forge pinning still pin to ipopt 3.14 that donwloads 3.14.9, but idyntree instead is pinned to ipot 3.14.8 . The problem is that migration https://conda-forge.org/status/#ipopt3148 is still not concluded.

**Comment by traversaro** [comment]
> The problem is that migration https://conda-forge.org/status/#ipopt3148 is still not concluded.

The migration is now concluded.
